charts/airflow/README.md

```diff
@@ -1130,6 +1130,30 @@ externalDatabase:
   properties: ""
 ```
 
+<h3>Option 3 - MsSQL</h3>
+
+> 🟨 __Note__ 🟨 
+> 
+> You must set `explicit_defaults_for_timestamp=1` in your MySQL instance, [see here](https://airflow.apache.org/docs/stable/howto/initialize-database.html)
```

Comment:
This warning is not related to MsSQL, but MySQL.

Also, we should specify the requirements that are listed on the ["setting up a MsSQL Database"](https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#setting-up-a-mssql-database) from the official airflow docs.  For example, about the requirement that `READ_COMMITTED_SNAPSHOT` be enabled on the DB.